src/cryptonote_core/blockchain.cpp

```diff
@@ -1056,17 +1068,17 @@ bool Blockchain::switch_to_alternative_blockchain(std::list<block_extended_info>
 
   // pop blocks from the blockchain until the top block is the parent
   // of the front block of the alt chain.
-  std::list<block> disconnected_chain;
+  std::list<block_and_checkpoint> disconnected_chain; // TODO(loki): use a vector and rbegin(), rend() because we don't have push_front
```

Comment:
This TODO doesn't really seem like something that will ever be worth actually bothering to do.

Reply:
Simple task I can give to junior if ever.